debian/rules

```diff
@@ -7,3 +7,6 @@ override_dh_auto_configure:
 	dh_auto_configure --                                                                              \
 	      -DCMAKE_CXX_FLAGS="-march=sandybridge -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer" \
 	      -DCMAKE_BUILD_TYPE=Release
+
+override_dh_auto_build:
+	dh_auto_build -- VERBOSE=1
```

Comment:
You could consider removing this again - no strong opinion on this though.

Reply:
Done.